mule-apikit-maven-plugin/src/main/java/org/mule/tools/apikit/model/ScaffoldingConfigurationMojo.java

```diff
@@ -3,6 +3,16 @@
 public class ScaffoldingConfigurationMojo {
     private boolean showConsole;
     private String externalCommonFile;
+    private String apiId;
+
+    public ScaffoldingConfigurationMojo(){
+        this.showConsole=true;
```

Comment:
Call the constructor with parameters to initialize the rest of the attributes, just as a good practice